docs/concepts/time-and-windows-in-ksqldb-queries.md

```diff
@@ -410,6 +410,189 @@ SELECT orderzip_code, TOPK(order_total, 5) FROM orders
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-zlqz{background-color:#c0c0c0;border-color:inherit;font-weight:bold;text-align:center;vertical-align:top}
+.tg .tg-gdc4{background-color:#96fffb;border-color:inherit;text-align:center;vertical-align:top}
+.tg .tg-4m7p{background-color:#9aff99;border-color:inherit;text-align:center;vertical-align:top}
+.tg .tg-84w4{background-color:#9aff99;border-color:inherit;font-weight:bold;text-align:center;vertical-align:top}
+</style>
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-zlqz">Message order</th>
+    <th class="tg-zlqz">order_zipcode</th>
+    <th class="tg-zlqz">Window_Start</th>
+    <th class="tg-zlqz">Window_End</th>
+    <th class="tg-zlqz">Window_Emit</th>
+    <th class="tg-zlqz">number_of_orders</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-gdc4">1</td>
+    <td class="tg-gdc4">94041</td>
+    <td class="tg-gdc4">12h00mn00s</td>
+    <td class="tg-gdc4">12h05mn00s</td>
+    <td class="tg-gdc4">12h00mn02s</td>
+    <td class="tg-gdc4">1</td>
+  </tr>
+  <tr>
+    <td class="tg-gdc4">2</td>
+    <td class="tg-gdc4">94041</td>
+    <td class="tg-gdc4">12h00mn00s</td>
+    <td class="tg-gdc4">12h05mn00s</td>
+    <td class="tg-gdc4">12h01mn23s</td>
+    <td class="tg-gdc4">2</td>
+  </tr>
+  <tr>
+    <td class="tg-gdc4">3</td>
+    <td class="tg-gdc4">94041</td>
+    <td class="tg-gdc4">12h00mn00s</td>
+    <td class="tg-gdc4">12h05mn00s</td>
+    <td class="tg-gdc4">12h03mn00s</td>
+    <td class="tg-gdc4">3</td>
+  </tr>
+  <tr>
+    <td class="tg-4m7p">4</td>
+    <td class="tg-4m7p">94041</td>
+    <td class="tg-84w4">12h05mn00s</td>
+    <td class="tg-84w4">12h10mn00s</td>
+    <td class="tg-84w4">12h05mn30s</td>
+    <td class="tg-4m7p">1</td>
+  </tr>
+  <tr>
+    <td class="tg-gdc4">5</td>
+    <td class="tg-gdc4">94041</td>
+    <td class="tg-gdc4">12h00mn00s</td>
+    <td class="tg-gdc4">12h05mn00s</td>
+    <td class="tg-gdc4">12h04mn00s</td>
+    <td class="tg-gdc4">4</td>
+  </tr>
+  <tr>
+    <td class="tg-4m7p">6</td>
+    <td class="tg-4m7p">98041</td>
+    <td class="tg-84w4">12h05mn00s</td>
+    <td class="tg-84w4">12h10mn00s</td>
+    <td class="tg-84w4">12h06mn10s</td>
+    <td class="tg-4m7p">2</td>
+  </tr>
+</tbody>
+</table>
+
+!!! note
+    On {{ site.cp }}, you can emit only the final result of the window by
+    using EMIT FINAL instead of EMIT CHANGES. In this case, you should set the
+    GRACE PERIOD explicitly, otherwise the window is emitted only after the
+    default grace period of 24 hours.
+
```

Comment:
Can we add another note : Note: frequency of updates can be fine tuned using [ksql.streams.commit.interval.ms](ksql.streams.cache.max.bytes.buffering) and [ksql.streams.cache.max.bytes.buffering](ksql.streams.cache.max.bytes.buffering) configurations.